packages/zapp/console/src/components/Literals/LiteralValue.tsx

```diff
@@ -31,6 +31,7 @@ export const LiteralValue: React.FC<{
             className={literalStyles.nestedContainer}
             map={literal.map as LiteralMap}
             showBrackets={true}
+            showJson={false} // todo: remove
```

Comment:
nit: can this todo be more descriptive?

Reply:
nit: uppercase for todo -> TODO:

Reply:
thanks, this is not intended to be checked in, i'll remove once we switch to json view.